### PR TITLE
Update primary button style

### PR DESCRIPTION
**Pull request by gtsiolis**
## Description

Reviving parts of https://github.com/gitpod-io/gitpod/pull/16433 to update the primary button style now that https://github.com/gitpod-io/gitpod/pull/18283 has been merged. 📙 

Besides the button color change, this will also increase the border radius to match the button radius used in the marketing website. Cc @chrifro @taliamoyal 

Additionally, this will update the theme selection card to avoid clashing with the onboarding.

<details>
<summary>Summary generated by Copilot</summary>

</details>

| BEFORE | AFTER |
|--------|--------|
| Frame 1745 | Frame 1746 |
| Frame 1743 | Frame 1744 |
| Frame 1761 | Frame 1762 |
| Frame 1763 | Frame 1764 |
| Frame 1747 | Frame 1748 |

## How to test

Log in and navigate through the dashboard to observe the new primary button style. 🐈‍⬛ 

#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - gt-update-b745e5f9b1</li>
	<li><b>🔗 URL</b> - <a href="https://gt-update-b745e5f9b1.preview.gitpod-dev.com/workspaces" target="_blank">gt-update-b745e5f9b1.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - gt-update-primary-button-style-gha.13852</li>
</ul>

## Build Options

<details>
<summary>Build</summary>

- [ ] /werft with-werft
      Run the build with werft instead of GHA
- [ ] leeway-no-cache
- [ ] /werft no-test
      Run Leeway with `--dont-test`
</details>

<details>
<summary>Publish</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer</summary>

- [ ] analytics=segment
- [ ] with-dedicated-emulation
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

<details>
<summary>Preview Environment</summary>

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [X] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
- [ ] with-monitoring
</details>

/hold

### CODE REVIEW

**Comment by chrifro** [comment]
Great change to align product and website more ✨

**Some thoughts:**
- nice border-radius 💫
- the orange seems quite pale, it's not the same color as we use on the website, is it?
- the disabled and enabled buttons are very similar and are difficult to distinguish. Would suggest adding a hover effect and more color difference to it
- FWIW, we just slightly updated the primary button for the website with a gradient style. Could look very cool in the product as well. See [here](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?type=design&node-id=513%3A527&mode=design&t=kKb3UdS5y9dhuVUi-1)

**Comment by gtsiolis** [comment]
Hey @chrifro! Thanks so much for the feedback.

> nice border-radius 💫

Thanks! 🙌 

> the orange seems quite pale, it's not the same color as we use on the website, is it?

Yeah, but the specific color looks quite good in dark theme, right?

For the light theme I still find it quite bad, but I just realized we're using the black background for such colors in light themes, and I was about to do just that, see screenshot below. 💡 

| Light | Dark |
|--------|--------|
| Frame 1765 | Frame 1766 | 

> the disabled and enabled buttons are very similar and are difficult to distinguish. Would suggest adding a hover effect and more color difference to it

I opted out from a hover state for now to avoid having too many kumquat colors around but yeah, I'll try something in this or a follow PR. Disabled buttons are a cancer to UX, and we should remove them all if possible. 🦀 

> FWIW, we just slightly updated the primary button for the website with a gradient style. Could look very cool in the product as well. See [here](https://www.figma.com/file/VL0CJpHqcDJVJ4BEkoyRFV/Gitpod-Website-Component-Library?type=design&node-id=513%3A527&mode=design&t=kKb3UdS5y9dhuVUi-1)

Awesome, I wanted to reach out about this. Superb. ✨

**Review by filiptronicek — APPROVED** [review]
This looks really good, great job with it, George. 

Not sure if you had anyone specific in mind for the review, but as for me, :shipit:.

**Comment by gtsiolis** [comment]
Thanks for taking a look, @filiptronicek. 🌮 

Made a few more minor changes:
- Align the log-in buttons to use the same border radius 🤓 
- Update some empty states background colors (if only we had a region component for this) 😃 

---

I'd like to have another pair of eyes on the UX changes.

@selfcontained any thoughts or strong objections against the colors used on light vs. dark theme? Cc @svenefftinge

File: components/dashboard/tailwind.config.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ module.exports = {
     theme: {
         extend: {
             colors: {
-                gray: colors.warmGray,
+                // gray: colors.warmGray,
```
Comment (selfcontained) [inline components/dashboard/tailwind.config.js:18]: let's remove this if we're good w/o it.

**Review by selfcontained — APPROVED** [review]
Changes lgtm. 

The only thing I noticed was the hover/active state of the primary button in the light theme was a bit too subtle for me to see.

File: components/dashboard/tailwind.config.js (hunk 1 above)
Comment (gtsiolis) [reply components/dashboard/tailwind.config.js]: Nice catch! ⚾ 

Rebased and removed.

**Comment by gtsiolis** [comment]
Thanks for taking a look, @selfcontained. 🌮 

Nice catch also on the subtle hover state of the button—for more context, this is by design to also mimic the subtle hover state of the button on the website. 💡 

Rebased and pushed.

Removing bold.

/unhold

**Pull request merged**